src/app.js

```diff
@@ -1,21 +1,33 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
 
-module.exports = sum;
+const PORT = process.env.port || 8080;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new url.URL(req.url, `http://${req.headers.host}`);
+  const myURL = normalizedURL.pathname.slice(1) || 'index.html';
```

Comment:
```suggestion
  const myURL = normalizedURL.pathname.replace('file/', '') || 'index.html';
```
and than at 15 line you can write only myURL. 
and than you can use !normalizedURl.pathname.startsWith('/file') instead of includes('file') and don`t put your fs.readFile into condition.